src/modules/search.js

```diff
@@ -143,9 +159,11 @@ class Search {
    * @param {string} [parameters.sortOrder='descending'] - The sort order for results
    * @param {string} [parameters.section='Products'] - The section name for results
    * @param {object} [parameters.fmtOptions] - The format options used to refine result groups
```

Comment:
Can we add `Please refer to https://docs.constructor.io/rest_api/search/queries for details`?

Reply:
Done 👍